src/components/LeafletMap.js

```diff
@@ -71,18 +66,19 @@ class LeafletMap extends Component {
       center: [38, 16],
       zoom: 2,
       minZoom: 3,
-      maxZoom: 11,
-      zoomControl: true
+      maxZoom: 14,
```

Comment:
The maximum zoom for this dataset at which you can see content from the map tiles is 13.

Reply:
Correction. Looks like this is just something that applies for Null Island. In SE Asia, this goes up to 17! 

Let's do 17